### ISSUE TITLE
yoyo and delay

### ISSUE THREAD

**Issue by vdaguenet** [legomushroom/mojs#15]
Hi!

I have an issue with a `Tween` that have a `delay` and `yoyo = true`. The repeat is called while the element  is coming back with the yoyo.

My code:
```js
var $square = document.querySelector('.square');

var t = new mojs.Tween({
  yoyo: true,
  repeat: 10,
  delay: 1000,
  duration: 800,
  onUpdate: function (progress) {
    $square.style.transform = 'translateX(' + progress * 100 + 'px)';
  }
});
t.run();
```

A demo is available here http://jsbin.com/fulebobodu/edit?html,css,js,output

Also with `repeat: 10`, I see 11 times my tween. It's a bit confusing.

**Comment by legomushroom** [comment]
Hi @vdaguenet!

Sorry for the delay, and thanks for the issue. That's definitely a `bug` and I do understand it's origin.
Will fix shorty.

Re: `repeat`.
The option describes how many times a tween should repeat `after it was completed`.
Tween will run once regardless the `repeat` option, so if you set repeat of `10`, it will run `11` (1 + 10) times in total.

Thanks for the alert!

**Comment by legomushroom** [comment]
Fixed with [[6c4cc6e](https://github.com/legomushroom/mojs/commit/6c4cc6e7e91c27cee9222e2b58701ea164234085)].
Will make release shortly.
